### ISSUE TITLE
Implement SmashcastChannel#getChatEmojis()

### ISSUE THREAD

**Issue by jens1o** [jens1o/smashcast-api-php#4]
https://developers.smashcast.tv/#get-emojis
- [x] returns an instance of SmashcastChatEmoji
- [x] Methods of `SmashcastChatEmoji`:
    - [x]  `__construct`: saves the name of the channel
    - [x] `getEmojis($premiumOnly = false)`: Returns the emojis for this channel, cached.

**Comment by jens1o** [comment]
Uploading/creating emojis is deferred, should be implemented after #5 has been implemented.

**Comment by jens1o** [comment]
I'm working on it, but at the moment, I'm learning for my last class tests...
